src/framework/mlt_property.c

```diff
@@ -821,7 +822,7 @@ char *mlt_property_get_string_l_tf( mlt_property self, locale_t locale, mlt_time
 		setlocale( LC_NUMERIC, localename );
 #endif // _WIN32
 
-		if ( self->types & mlt_prop_int )
+		if ( self->types & mlt_prop_int || self->types & mlt_prop_color )
```

Comment:
Continuing the conversation from the previous review comment...

> ... a hex string is not the preferred or typical format. CSS color string is.

I agree with that comment. There are many places in the code that use hex strings which I think we could convert to CSS style in the future:
https://github.com/mltframework/mlt/blob/master/src/modules/qt/filter_audiospectrum.cpp#L349
But that is for another day.

For this line of code specifically, I think we should return a HTML/CSS color code if someone requests a string on a property that was previously set as a color type. I would like to see that change  before we merge so that behavior does not need to  be changed in the future and possibly break compatibility.

Reply:
Okay, will look into this.

Reply:
Looking in to this again, I am really confused by the fact that MLT seems to use "wrong" CSS color strings in the pattern `#AARRGGBB` while correct CSS hex color strings have the alpha channel in the last position like `#RRGGBBAA`. Unfortunately I guess there is no way to change this behavior since it would break things, right?

Reply:
Okay, https://www.educative.io/answers/what-is-the-difference-between-rgba-and-argb somehow explains it. However it is still confusing.

Reply:
As best as I can recall at the time this was implemented CSS did not support an alpha component, and made  compatible with QColor: https://doc.qt.io/qt-6/qcolor.html

Reply:
Requesting a string gives you a `#AARRGGBB` string now, all covered by tests